ui/css/module/_footnotes.scss

```diff
@@ -0,0 +1,18 @@
+.node-footnote {
+  display: none;
+}
```

Comment:
Let's not hide these in CSS. If we don't want to display them, we shouldn't render them.